src/components/Form/CheckAclPermsDocContainerForm.jsx

```diff
@@ -108,12 +108,12 @@ const CheckAclPermsDocContainerForm = () => {
             <TextField
               id="set-acl-to"
               name="setAclTo"
-              value={username}
+              value={selectedUser.person ? selectedUser.person : username}
```

Comment:
Almost, it should be `value={selectedUser.person ? selectedUser.username : username}` since selectedUser.person will give the person's name instead of the person's username.

Reply:
It seems like the same line was used for all 5 forms, but the fix should be pretty quick.